src/classes/i18n4Js.php

```diff
@@ -51,6 +51,8 @@ private function getTerms(): array
             'request-filename' => _('Enter name of the file'),
             'save' => _('Save'),
             'saved' => _('Saved'),
+            'sort-column' => _('Sort column'),
+            'sort-by-column' => _('Sort by column'),
```

Comment:
maybe only one translated string would cover both case, no? Seems wasteful to have to translate these two very similar strings

Reply:
Yes, I will change it.